pages/login.vue

```diff
@@ -19,14 +18,14 @@
         url.searchParams.append('scopes', scopes.join(" "));
         url.searchParams.append('code_challenge_method', 'S256');
         url.searchParams.append('code_challenge', pkce.code_challenge);
-        navigateTo(url.toString(), {external: true});  // page redirect
+        navigateTo(url.toString(), {external: true});
     }
 
     async function loadToken(){
-        // generated in login()
-        const code_verifier = localStorage.getItem('code-verifier');
-
+        // login() only returns a code challenge. This function turns the code challenge into an auth-token
         const url = new URL('/api/token', 'https://accounts.spotify.com/');
+        const redirect_uri = window.location.href.replace("/\/login/", "/\/home/"); // this parameter needs to approved in the Spotify Developer Dashboard
```

Comment:
das klappt nicht, bei beide 'redirect_uri' müssen die gleiche sein. das ist eine weitere form der verifizierung
du kannst nicht oben const `redirect_uri = new URL(window.location);` und später eine modifizierte version machen. außerdem soll die auth logik nicht in /home oder so ein. das einzig akzeptable wäre wenn wir einen weiteren endpunkt wir /auth machen oder so welcher den code von loadToken beinhaltet